packages/server/src/integrations/utils.ts

```diff
@@ -184,7 +184,9 @@ export function getSqlQuery(query: SqlQuery | string): SqlQuery {
   }
 }
 
-export const isSQL = helpers.isSQL
+export function isSQL(datasource: Datasource) {
```

Comment:
What is the reason for this?

Reply:
For whatever reason I was getting IDE errors with typing around this - it wasn't fully understanding the typing of shared-core utilities. Will try to look into that separately, but this utility is quite important/useful and its nice to be able to see its typing easily without needing to jump into shared-core.